docs/setup.asciidoc

```diff
@@ -20,13 +20,12 @@ To get Kibana up and running:
 
 . Download the https://www.elastic.co/downloads/kibana[Kibana 4 binary package] for your platform.
 . Extract the `.zip` or `tar.gz` archive file.
+. After installing, run Kibana from the install directory: `bin/kibana` (Linux/MacOSX) or `bin\kibana.bat` (Windows).
 
-// On Unix, you can instead run the package manager suited for your distribution.
-//
-// [float]
-// include::kibana-repositories.asciidoc[]
-//
-After installing, run Kibana from the install directory: `bin/kibana` (Linux/MacOSX) or `bin\kibana.bat` (Windows).
+On Unix, you can instead run the package manager suited for your distribution.
+
+[float]
+include::kibana-repositories.asciidoc[]
```

Comment:
I know you and I spoke about this, so I'm just putting a note here so I don't forget about it: the repositories instructions should go out for alpha1, but they should at least be commented out in master.